src/utils/axiosFunctions.jsx

```diff
@@ -4,12 +4,17 @@ const backendAPI = axios.create({
   baseURL: "https://nc-news-app.onrender.com/api",
 });
 
-export function getArticles(topic) {
-  let APIstring = `/articles`;
-  if (topic && topic != "all") {
-    APIstring += `?topic=${topic}`;
-  }
+export function getArticles([
+  topic = "all",
+  sortby = "created_at",
+  order = "desc",
+]) {
+  let APIstring =
+    topic === "all"
+      ? `/articles?sortby=${sortby}&order=${order}`
+      : `/articles?topic=${topic}&sortby=${sortby}&order=${order}`;
 
```

Comment:
I like that you've further refactored the `getArticles` function here, rather than make a separate function 🙌 
To make the query adding functionality you're manually adding here a bit more succinct - consider using the [axios params object ](https://notes.northcoders.com/courses/js-front-end/axios#axios-params)